HelpSource/Classes/Object.schelp

```diff
@@ -200,6 +200,113 @@ A String containing the resulting file's path.
 
 subsection::Equality and Identity
 
+Equally asks if the value of two objects is equal.
+Identity asks if they are the same object.
+There are a few classes of object where only one instance exists and is shared between all uses (a singleton).
+These objects include Symbols and numbers.
+
+Equality, tested for with code::==:: or code::!=:: is a relation between objects.
+So code::a == b:: is true when the object in the variable a and the object in b are equivalent in some essential aspect.
+Two strings, for example, are equal when they have the same characters in the same order.
```

Comment:
I'd put the code example immediately under this.
```
"we are all the same" == "we are all the same" // true
"we are all the same" === "we are all the same" // false: same, but not identical
```

Reply:
Thanks I will do. See https://github.com/supercollider/supercollider/pull/6385#discussion_r1667346159